uint8 STATUS_STOPPED=0
uint8 STATUS_MANUAL=1
uint8 STATUS_AUTONOMOUS=1

std_msgs/Header header
uint8 status
float32 battery_voltage
